### ISSUE TITLE
--ci not adding new files

### ISSUE THREAD

**Issue by spirillen** [funilrys/PyFunceble#216]
## Bug description
Without adding the `--ci-command "git add ."` nothing is committed when running under CI/CD tested in 3 repos now

https://github.com/mypdns/matrix/blob/a23578592bf4bbc61948190e5cca1b9b8d7126c2/.github/workflows/pyfunceble.yml#L70

**Comment by funilrys** [comment]
Hey @spirillen,

thank you for reporting this.

But, can you explain why it is working in our [ci_test](https://github.com/PyFunceble/ci_test) repository?

**Comment by spirillen** [comment]
If I could, I could be rich :money_mouth_face: 

The only thing I can link (In time) is this commit https://github.com/funilrys/PyFunceble/commit/2f57772c5518ccd32afdb79dcfa1557122867c5e (Fix issue with the session ID. ), which solved one of the commit issue and then it kind a broke down again for the three others and with https://github.com/funilrys/PyFunceble/commit/2a583fc12441e9bb18a6edf3ac026032f531990f (Fix issue with preloading.) we got the preloading working. Haven't seen anything wrong here except that I started to use the `--ci-command` to have it committed

And when I'm reading the code I don't see why this should happen, unless it is happening a hole different place apart from the git handler.

I honestly don't even having a got feeling, clues or what ever for where this should occur

However, there is another issue we might take a look at, speed, without the `--preload` my test files are simply to big (pyfunceble to slow) to even start a test within 45 minutes. and that is a huge problem as I would prefer running all clients from same sql DB, but that is another thread, would just mention it as I noticed it yesterday on two of the repos
